requirements.txt

```diff
@@ -1,4 +1,13 @@
+asgiref==3.7.2
+autopep8==2.0.4
+Django==5.0.1
+django-cors-headers==4.3.1
+djangorestframework==3.14.0
+evagram==0.0.0

```

Comment:
We should avoid having evagram be a requirement to evagram. Can you remove this and have the github action install evagram before running tests (if evagram needs to be installed)?

Reply:
Sure, I can do that